client/components/accordion/status.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import Tooltip from 'components/tooltip';
+
+/**
+ * Constants
+ */
+const STATUS_GRIDICON = {
+	info: 'info',
+	warning: 'notice',
+	error: 'notice'
+};
+
+export default class AccordionStatus extends PureComponent {
+	static propTypes = {
+		type: PropTypes.oneOf( [
+			'info',
+			'warning',
+			'error'
+		] ),
+		text: PropTypes.node,
+		url: PropTypes.string,
+		position: PropTypes.string,
+		onClick: PropTypes.func
+	};
+
+	static defaultProps = {
+		type: 'info',
+		onClick: () => {}
+	};
+
+	state = {};
```

Comment:
Or, if the question is whether we need to initialize it as we have here, then yes, because by default a component's `state` is `undefined`. We could account for this in `render` but I think it's useful as a reference that we do use state in the component (maybe even more useful if we included the keys that will ever exist, even if `null` to begin with).